exo10h-12h.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>exercises10h-12h</title>
+    <script>
+        function fact(nbr){
+            if(nbr === 0)
+            {
+                return 1;
+            }
+            return nbr * fact(nbr-1);
+        }
+        console.log(fact(3));
+
+        function pgcd(a,b){
+           let r=a;
+           do {
+               a = b;
+               b = (r % a)/2;
+           } while(b>0);
+
+           return a
+       }
+      console.log(pgcd(45,50));
+
+       function fibonacci(n){
+           if(n < 2){
+
+           return n;
```

Comment:
You need to return an array of numbers not just one number. By the way, fib of 2 is not 2 and fib of 1 is not 1 and fib of 0 is not 0